### ISSUE TITLE
Progress not showing when updating docset

### ISSUE THREAD

**Issue by PhilT** [zealdocs/zeal#707]
Tried with both 0.3.1 stable and latest master running on Windows 10 x64. Trying to update Mono.

Downloads 100% but then looks like it's complete when in fact it's actually installing the new docset. I get the message "Please wait for all operations to complete" if I try to close the dialog until it's finished installing (confirmed by checking CPU and disk activity (seems to work a lot quicker with Defender disabled too).

**Comment by hugodahl** [comment]
As an additional related progress issue (I believe), is that when a docset update has been requested, the list still shows "Update Available" and leaves the "Update" button enabled, leading one to believe that the update button wasn't actually/properly clicked in the first place.

**Comment by trollixx** [comment]
Yes, that's a bug as well. Clicking update twice will lead to a crash.

**Comment by AyrA** [comment]
I want to add here that this still is the case. Some documentations are huge and a progress indicator for them would certainly be helpful.
